### ISSUE TITLE
Makefile delete all my files from mac

### ISSUE THREAD

**Issue by SergeyOleynich** [Carthage/Carthage#1655]
* `carthage version`: 0.18.1
* `xcodebuild -version`: Xcode 8.1
* Are you using `--no-build`? no
* Are you using `--no-use-binaries`? no
* Are you using `--use-submodules`? no

**Cartfile**
Download zip. Open Makefile. Result delete all my files from my mac.

**Carthage Output**
What I should do? Why it is so?

**Comment by mdiep** [comment]
You've given no information about what you did, so I'm not sure how to help.

**Comment by SergeyOleynich** [comment]
ok. I download the zip from main page. Unzip it. after this in spotlight I wrote this Carthage.pkg and spotlight showed me one file and I double click ob it. At first it did not open and I opened settings -> security and choose open file. After this terminal started to work. After some time it asked me my pass in order to go further. After I think 5 or 7 minutes the process was ended and I restart my mac. After this it asked my iCloud account and start to configure my mac. When it launched I have empty hard drive.
